org.eclipse.xtext.xbase/src-gen/org/eclipse/xtext/xbase/services/XbaseGrammarAccess.java

```diff
@@ -96,21 +99,23 @@
 		//XAssignment
 		public RuleCall getValueXAssignmentParserRuleCall_0_3_0() { return cValueXAssignmentParserRuleCall_0_3_0; }
 		
-		//XOrExpression (=> ({XBinaryOperation.leftOperand=current} feature=[types::JvmIdentifiableElement|OpMultiAssign])
-		//rightOperand=XAssignment)?
+		//XOrExpression (
+		//    =>({XBinaryOperation.leftOperand=current} feature=[types::JvmIdentifiableElement|OpMultiAssign]) rightOperand=XAssignment
+		//)?
 		public Group getGroup_1() { return cGroup_1; }
 		
 		//XOrExpression
 		public RuleCall getXOrExpressionParserRuleCall_1_0() { return cXOrExpressionParserRuleCall_1_0; }
 		
-		//(=> ({XBinaryOperation.leftOperand=current} feature=[types::JvmIdentifiableElement|OpMultiAssign])
-		//rightOperand=XAssignment)?
+		//(
```

Comment:
Here is one of the rare cases that works not so well. But I would consider it minor.

Reply:
Agreed